content/blog/2023-03-02-ethereum-classic-course-11-ethereum-classic-social-and-physical-layer-security/index.md

```diff
@@ -0,0 +1,96 @@
+
+But, how do the security models in each layer work? And, how do they work combined?
+
+## 3. Physical Layer: Proof of Work Security
+
+![The physical layer.](./2.png)
+
+As we mentioned before, proof of work is the key security component of the physical layer, or live operating network.
+
+The way it works is that there must be demand for ETC’s decentralized computing. If there is demand, then people will buy the cryptocurrency to use it, and as an investment, making the price rise. When the ETC price rises, then mining becomes more profitable. When miners becomes more profitable, they put more machines with more hashing power to build blocks for the network. When more hashing power is dedicated to the network, then it becomes more secure. And, when ETC is more secure, then demand for its decentralized computing grows, which, in turn, makes the price go up further, continuing the virtuous circle.
+
+## 4. Social Layer: Coordination Problem Security
+
+![The social layer.](./3.png)
+
+On the social layer, the key security component is that for all the constituents who participate in the network (i.e. exchanges, wallets, dapps, startups, Node as a Service (NaaS) providers, block explorers, independent participants, mining pools, and independent miners); who reside in different parts of the world, in different cultures, with different languages, and different worldviews; it is extremely difficult to coordinate to attack the network, censor it, or corrupt it. 
+
+However, the economic incentives to use the network are conducive of increased use, as demand for decentralized computing grows. If demand for decentralized computing grows, then more dapp developers will build useful things on ETC. As more useful dapps will exist on the blockchain, then more users will be attracted to it. If more users are attracted to it, then more node operators will participate. The more node operators participate, from different parts of the world, the more difficult it is to coordinate attacks, making the blockchain even more secure. And, when ETC is more secure, then demand for its decentralized computing grows, which in turn makes more dapp developers build on ETC, continuing the virtuous circle.
+
+## 5. The ETC Security Model Combined
+
+![The ETC security model combined.](./4.png)
+
+The way these two security virtuous circles work combined; with proof of work as the key component at the live operating network, and the coordination problem as the key component at the social layer; is that both have incentives to increase price and participation, and as both sides grow, then demand for ETC computing services grows, which increases price and participation even more.
```

Comment:
Line 61: Rewrite for clarity
```suggestion
These two virtuous circles of security work with Nakamoto consensus proof of work as the key component of the live operating network and the difficulty of attack coordination as the key component of the social layer so that both layers give incentive to increasing price and participation. As both sides grow, demand for ETC computing services grows which increases price and participation even more.
```